File: src/aderm/me/commissary/events/Joins.java
package aderm.me.commissary.events;

import org.bukkit.entity.Player;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerJoinEvent;

import aderm.me.commissary.Main;

import java.io.IOException;

public class Joins implements Listener {

    private Main main;

    public Joins(Main pl) {
        main = pl;
    }

    @EventHandler
    public void onJoin(PlayerJoinEvent e) {
        Player p = e.getPlayer();
        if (!(main.players.contains(p.getUniqueId().toString()))) {
            main.players.set(p.getUniqueId().toString() + ".Points", Integer.valueOf(0));

            try {
                main.players.save(main.playersyml);
            } catch (IOException i) {
                i.printStackTrace();
            }

        }
    }

}

File: src/me/deceptions/commissary/events/Joins.kt
package me.deceptions.commissary.events

import me.deceptions.commissary.Main
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerJoinEvent
import java.io.IOException

class Joins(private val main: Main) : Listener {

    @EventHandler
    fun onJoin(e: PlayerJoinEvent) {
        val p = e.player
        if (!main.players.contains(p.uniqueId.toString())) {
            main.players.set(p.uniqueId.toString() + ".Points", Integer.valueOf(0))

            try {
                main.players.save(main.playersyml)
            } catch (i: IOException) {
                i.printStackTrace()
            }

        }
    }

}